docs/docs/index.md

```diff
@@ -15,278 +15,129 @@
-
-    !!! note
-        Be sure to run Jupyter from this virtual environment, not from
-        another installation, or the `rsconnect-python` extension will
-        not be available. To do so, you will need to activate the virtual
-        environment in each new terminal session before you run `jupyter`.
-
-## Upgrading
-
-To upgrade `rsconnect-jupyter`:
-
-- First, uninstall the extension.
-- Then, re-install it.
-
-## Usage
-
-To publish to RStudio Connect:
-
-- Open a notebook.
-- Click the <img class="icon"; src="images/publish-icon.gif" alt="blue toolbar icon used for publishing the notebook"> icon (blue publish icon) and select `Publish to RStudio Connect`
-to publish the current notebook to RStudio Connect.
-
-### Entering server information
-
-- If this is your first time publishing a notebook, you will be
-prompted to enter the location and a nickname for the RStudio Connect server.
-- You will also be prompted to enter your API Key. See the [RStudio Connect User
-Guide](http://docs.rstudio.com/connect/user/api-keys.html) for
-instructions on generating API Keys for your user.
-- When you click the **Add Server** button, `rsconnect-jupyter` will send a request to the RStudio Connect server to verify that it can be reached via the requested URL and that the API key is valid.
-
-If your RStudio Connect server was configured with a self-signed certificate (or other certificate that computer hosting your Jupyter notebook server does not trust), the attempt to contact RStudio Connect may fail with a TLS-related error.
-
-You have multiple options in this case, depending on your needs:
-
-1. If your RStudio Connect Administrator can give you the Certificate Authority (CA)
- Bundle for your RStudio Connect server, ask your Jupyter Administrator if it
- can be added to the trusted system store.
-1. If the CA Bundle cannot be added to the trusted system store, you may select
- `Upload TLS Certificate Bundle` to upload the bundle to Jupyter, which will verify
- your secure connection to RStudio Connect.
-1. If you cannot obtain the CA bundle, you can disable TLS verification completely
- by selecting the `Disable TLS Certificate Verification` box. Your connection to
- RStudio Connect will still be encrypted, but you will not be able to verify the
- identity of the RStudio Connect server.
+    The `rsc-jupyter` package is developed for Jupyter Notebook, specifically. Therefore, the package does not work with the JupyterLab development environment.
 
-<img src="images/add-dialog.png" class="block" alt="initial dialog that prompts for the location of RStudio Connect">
+This documentation covers three methods:
 
-### Publishing options
+- [Installation in JupyterHub](#installing-in-jupyterhub)
+- [Installing `rsconnect-jupyter` to Jupyter running on RStudio Workbench](#installing-to-jupyter-running-on-rstudio-workbench)
+- [Installing Jupyter within a virtual environment](#installing-jupyter-within-a-virtual-environment)
```

Comment:
The most common case is probably installation in a virtual environment, then in RStudio Workbench. JupyterHub installation is done once by an administrator and is the most complex section. I would suggest reordering these so JupyterHub is last.

Reply:
Not certain why this isn't showing as outdated but I did push these changes